seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/phoenix/PhoenixDialectFactory.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.phoenix;
+
+import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.JdbcDialect;
+import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.JdbcDialectFactory;
+import org.apache.seatunnel.connectors.seatunnel.jdbc.internal.dialect.mysql.MysqlDialect;
+
+import com.google.auto.service.AutoService;
+import lombok.NonNull;
+
+@AutoService(JdbcDialectFactory.class)
+public class PhoenixDialectFactory implements JdbcDialectFactory {
+
+    @Override
+    public boolean acceptsURL(@NonNull String url) {
+        // Support greenplum native driver: com.pivotal.jdbc.GreenplumDriver
+        return url.startsWith("jdbc:phoenix:");
+    }
+
+    @Override
+    public JdbcDialect create() {
+        return new MysqlDialect();
```

Comment:
Since I haven't used Phoenix, I'm not quite sure that Phoenix can use the mysql dialect directly? For example, the Time and Blob types of Phoenix consistent with MySQL?

Reply:
> Since I haven't used Phoenix, I'm not quite sure that Phoenix can use the mysql dialect directly? For example, the Time and Blob types of Phoenix consistent with MySQL?
Thanks for your suggestion. I re-implemented PhoenixTypeMapper.
Refer to the https://phoenix.apache.org/language/datatypes.html.

Reply:
> Since I haven't used Phoenix, I'm not quite sure that Phoenix can use the mysql dialect directly? For example, the Time and Blob types of Phoenix consistent with MySQL?

Thanks for your suggestion. I re-implemented PhoenixTypeMapper.
Refer to the https://phoenix.apache.org/language/datatypes.html.